lib/cmds/update.ts

```diff
@@ -150,13 +154,13 @@ function update(options: Options): Promise<void> {
     updateBrowserFile(binary, outputDir);
     promises.push(updateBinary(binary, outputDir, proxy, ignoreSSL));
   }
-  if (ie) {
+  if (ie64) {
     let binary = binaries[IEDriver.id];
     binary.arch = Config.osArch();  // Win32 or x64
     updateBrowserFile(binary, outputDir);
     promises.push(updateBinary(binary, outputDir, proxy, ignoreSSL));
   }
-  if (ie32) {
+  if (ie || ie32) {
```

Comment:
ie, ie32, and ie64 are set from different flag options.

An example: a user downloads the 32bit version:
```
webdriver-manager update --ie

ie = true
ie32 = false
ie64 = false

webdriver-manager update --ie32

ie = false
ie32 = true
ie64 = false
```

An odd case: the user wants to download the 32bit version and 64bit version. We will let them download both. However, starting the server, the user would then have to specify if they want to use the 32-bit version if not it would default to the system arch.
```
webdriver-manager update --ie32 --ie64

ie = false
ie32 = true
ie64 = true
```